dialogue-apache-hc4-client/build.gradle

```diff
@@ -1,5 +1,4 @@
 apply from: "$rootDir/gradle/publish-jar.gradle"
-apply plugin: 'com.palantir.revapi'
```

Comment:
why the deletion here? i'd prefer to just use `./gradlew revapiAcceptAllBreaks` and start building up a revapi.yml of accepted breaks?

Reply:
I picked the wrong number of commits in `git checkout HEAD~4 -- dialogue-apache-hc4-client` :-( fixed.